doc/source/testing.rst

```diff
@@ -115,26 +115,32 @@ Ideally, you want all tests to pass. This will look like:
 
     .. image:: images/ci_checks_passed.png
 
-.. _appveyor:
+.. _github-actions:
 
---------
-Appveyor
---------
+--------------
+GitHub Actions
+--------------
+
+Configured in `.github/` directory. Uses YAML syntax to define both workflows and actions. See `GitHub Actions documentation <https://help.github.com/en/actions>`_ for more information.
+
+
+.. _azure:
 
-`AppVeyor`_ is a continuous integration/continuous deployment service. MDAnalysis uses it for `testing builds on Windows`_.
+-----
+Azure
+-----
 
-Builds are configured in the file ``.appveyor.yml``. If you add a new dependency to MDAnalysis, you will need to add it to the ``$CONDA_DEPENDENCIES`` or ``$PIP_DEPENDENCIES`` in ``.appveyor.yml`` to pass tests.
+Configured in `azure-pipelines.yml` file. Uses YAML syntax to define Azure Pipelines tasks. See `Azure Pipelines documentation <https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema>`_ for more information.
 
-.. _`testing builds on Windows`: https://ci.appveyor.com/project/orbeckst/mdanalysis
 
+.. _cirrus:
 
-.. _travis:
+---------
+Cirrus CI
+---------
 
-------
-Travis
-------
+Configured in `.cirrus.star` file. Uses the Starlark syntax in YAML to define tasks. See `Cirrus CI documentation <https://cirrus-ci.org/guide/writing-tasks/>`_ for more information.
```

Comment:
If we're going with this can we at least point out to the fact that the actual files aren't there? .cirrus.star is just a "what do I do" thing